subsys/testsuite/coverage/coverage_ram.ld

```diff
@@ -31,7 +31,21 @@ __gcov_bss_num_words = ((__gcov_bss_end - __gcov_bss_start) >> 2);
 __gcov_bss_size = __gcov_bss_end - __gcov_bss_start;
 #endif
 
-#ifdef CONFIG_X86
+#ifdef CONFIG_X86_64
```

Comment:
I assume we cannot combine those 2 sections, right?

Reply:
@finikorg  thanks for you review. Yes, I think we better split them up.